src/admin/content-page/components/blocks/MediaGridWrapper/MediaGridWrapper.tsx

```diff
@@ -318,6 +320,9 @@ const MediaGridWrapper: FunctionComponent<MediaGridWrapperProps & UserProps> = (
 			itemOrCollectionOrAssignment?.view_counts_aggregate?.aggregate?.sum?.count || 0;
 
 		const element: MediaGridBlockComponentState = (elements || [])[index] || ({} as any);
+		const showCopyrightNotice =
+			!!itemOrCollectionOrAssignment.copyrightOrganisation &&
+			(!commonUser || location.pathname.startsWith(ADMIN_PATH.DASHBOARD));
```

Comment:
Avoid using `location`, use react-router's `useLocation` hook instead.

Reply:
ok, switched to withRouter instead of window.location